mautic/mautic
# Transactional emails in campaigns ignore the DNC setting

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR suggets ignoring the DNC setting the user might have done for a transactional email (e.g. password forget, login info, etc.). 


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Set "Do Not Contact" in contacts view.
3. Create a new campaing to send an email to the contact.
4. Check if the email has been received.
5. Send the email to the contact `POST /emails/ID/contact/CONTACT_ID/send`.
6.  Check if the email has been received.



4. Check if the email has been received.
